modules/tinymce/src/themes/silver/main/ts/Autocompleter.ts

```diff
@@ -50,34 +51,17 @@ const register = (editor: Editor, sharedBackstage: UiFactoryBackstageShared) =>
   );
 
   const isMenuOpen = () => InlineView.isOpen(autocompleter);
-  const isActive = () => activeAutocompleter.get().isSome();
+  const isActive = () => activeState.get();
```

Comment:
```suggestion
  const isActive = activeState.get;
```
This can be simplified/reduced to this